marlowe-playground-client/src/Simulation/BottomPanel.purs

```diff
@@ -318,15 +331,43 @@ panelContents state MarloweErrorsView =
     ]
     content
   where
-  errors = state ^. (_marloweState <<< _Head <<< _editorErrors)
+  errors = state ^. (_marloweState <<< _Head <<< _editorErrors <<< to (map formatError))
```

Comment:
Instead of `to (map f)`, `traversed <<< to f` would be more standard.

Reply:
too fiddley because I have to use `^..` which returns a `List` rather than an `Array` which I would then have to convert to an `Array` for the `HTML` code.